gamemode/modules/hobo/sv_hobo.lua

```diff
@@ -1,21 +1,37 @@
+local sounds = {
+    {
+        range = 14,
+        path = "npc/zombie/zombie_voice_idle"
+    },
+    {
+        range = 6,
+        path = "npc/zombie/zombie_pain"
+    },
+    {
+        range = 3,
+        path = "npc/zombie/zombie_alert"
+    }
+}
+
 local function MakeZombieSoundsAsHobo(ply)
-    if ply:EntIndex() == 0 then return end
+    if ply:EntIndex() == 0 then 
+        return 
```

Comment:
Trailing whitespace on both lines. With most modern editors you can turn on a setting to "trim trailing whitespace", which automatically removes spaces and tabs at the end of lines upon saving a file.